app/latigo/gordo/gordo_exceptions.py

```diff
@@ -5,7 +5,9 @@ class NoTagDataInDataLake(Exception):
     """Raise if there's no data in the Data Lake for particular tag for given period of time."""
 
     def __init__(self, project_name, model_name, from_time, to_time, e):
-        self.message = f"Prediction was skipped. Reason: no data was found in the Data Lake for " \
-                       f"project='{project_name}' :: model='{model_name}' :: " \
-                       f"from='{from_time}' :: to='{to_time}' :: error='{e}'"
+        self.message = (
+            f"Prediction was skipped. Reason: no data was found in the Data Lake for "
```

Comment:
I think "f" is not needed here
```suggestion
            "Prediction was skipped. Reason: no data was found in the Data Lake for "
```